pkg/cmd/cli/describe/projectstatus_test.go

```diff
@@ -40,7 +40,7 @@ func TestProjectStatus(t *testing.T) {
 						Name:      "example",
 						Namespace: "",
 						Annotations: map[string]string{
-							"displayName": "Test",
+							"openshift.io/displayName": "Test",
```

Comment:
the tests can inline the strings, rather than reference the constants, because we want tests to break if we change a constant people have coded against